runtime/compiler/CMakeLists.txt

```diff
@@ -115,6 +115,19 @@ if(JITSERVER_SUPPORT)
 	)
 
 	add_custom_target(j9jit_proto DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.h ${CMAKE_CURRENT_SOURCE_DIR}/net/gen/compile.pb.cpp)
+
+	if(OPENSSL_BUNDLE_LIB_PATH) # --enable-openssl-bundling
+		set(OPENSSL_ROOT_DIR ${OPENSSL_BUNDLE_LIB_PATH})
+		SET(CMAKE_SHARED_LINKER_FLAGS "-Wl,-rpath,$ORIGIN/.. ${CMAKE_SHARED_LINKER_FLAGS}")
```

Comment:
Why does this not append (rather than prepend) like the makefile version?

Reply:
No particular reason. Will do append.